CHANGELOG.MD

```diff
@@ -6,6 +6,7 @@
 * Retrieve updated-by user detail en-masse. Refs UIPOLICIES-11.
 * Add `tenantId` and `affiliationSelectionComponent` props to support Authorization Policies in Consortium settings. Refs UIPOLICIES-14.
 * Add Deque Axe a11y tests. Refs UIPOLICIES-19.
+* *BREAKING* migrate react-intl to v7. Refs UIPOLICIES-22.
```

Comment:
Since this is breaking, this release should be bumped to 2.0.0 here and in package.json.